specifications/move_adapter/README.md

```diff
@@ -2,36 +2,50 @@
 
 ## Introduction
 
-The **Move Adapter** is responsible for the validation and execution of transactions.
-**Validation** and **Execution** are different steps, exposed through different entry points
-and using different instances of the adapter.
-
-_[Validation](#Validation)_ performs various checks on a transaction to determine if it is well formed and to evaluate its priority.
-When entering the system, each user transaction is validated relative to a starting state and either discarded if it is malformed or added to a ranked pool of pending transactions otherwise.
-A **discarded** transaction is removed from the system without being recorded on the blockchain.
-A **validated** transaction can proceed but may still be discarded during the execution step.
-
-_[Execution](#Execution)_ receives a block of transactions and a starting state and evaluates those transactions in order.
-Each transaction is first revalidated. The system state may have changed since the validation step, so that a transaction has become invalid and needs to be discarded.
-Otherwise, the execution process computes a set of side effects for each transaction. Those side effects are used in two ways: they are included in the final per-transaction results of the execution step, and they are applied locally within the execution process so that they are visible to subsequent transactions within the same block.
+The **Move Adapter** is responsible for the validation and execution of
+transactions. **Validation** and **Execution** are different steps, exposed
+through different entry points and using different instances of the adapter.
+
+_[Validation](#Validation)_ performs various checks on a transaction to
+determine if it is well formed and to evaluate its priority. When entering
+the system, each user transaction is validated relative to a starting state
+and either discarded if it is malformed or added to a ranked pool of pending
+transactions otherwise. A **discarded** transaction is removed from the
+system without being recorded on the blockchain. A **validated** transaction
+can proceed but may still be discarded during the execution step.
+
+_[Execution](#Execution)_ receives a block of transactions and a starting
+state and evaluates those transactions in order. Each transaction is first
+revalidated. The system state may have changed since the validation step, so
```

Comment:
```suggestion
re-validated. The system state may have changed since the validation step, so
```

Reply:
OK, this could go either way, but I don't mind adding hyphens.